build-parent/pom.xml

```diff
@@ -81,13 +81,13 @@
         <junit-pioneer.version>2.2.0</junit-pioneer.version>
 
         <!-- Database images for JDBC/Reactive/Hibernate tests and devservices -->
-        <postgres.image>docker.io/postgres:14</postgres.image>
+        <postgres.image>docker.io/postgres:16</postgres.image>
```

Comment:
LGTM, that's what Hibernate ORM 6.5 gets tested against: https://github.com/hibernate/hibernate-orm/blob/6.5/docker_db.sh#L190-L192